### PR TITLE
[WIP] Update Crystal and frameworks

### PR DESCRIPTION
**Pull request by vlazar**
Updates Crystal and all Crystal frameworks to the latest versions.

### CODE REVIEW

File: frameworks/Crystal/amber/src/controllers/benchmark_controller.cr [hunk 1]
```diff
@@ -26,7 +26,7 @@ class BenchmarkController < Amber::Controller::Base
     response.content_type = JSON
     results = {} of Symbol => Int32
     if world = World.find rand(1..ID_MAXIMUM)
-      results = {id: world.id, randomNumber: world.randomNumber}
+      results = {id: world.id, randomNumber: world.randomnumber}
```
[the remark this answers is not in the document]
Comment (vlazar) [reply frameworks/Crystal/amber/src/controllers/benchmark_controller.cr]: It sure is. The column in DB is named `randomnumber` however.

[the remark this answers is not in the document]
Comment (faustinoaq) [reply frameworks/Crystal/amber/src/controllers/benchmark_controller.cr]: Oh, then `randomnumber` is ok :+1:

**Comment by nbrady-techempower** [comment]
Kemal is failing here where it's passing in other runs. Any ideas as to why these updates might break that?

**Comment by michaelhixson** [comment]
@nbrady-techempower The submitter was following up on that in kemal's issues list:  https://github.com/kemalcr/kemal/issues/459

It sounds like changes in Crystal were incompatible with something the kemal implementation is doing, but there may be a workaround.  I'm expecting that this PR will eventually be updated with one of these fixes:
* Add the workaround in the kemal test implementation
* Update the version of the kemal framework to one where the workaround isn't necessary (not sure if this is being worked on)
* Undo the change to the version of crystal that the kemal implementation is using

I don't think we should merge it in this state while kemal is failing.

**Comment by vlazar** [comment]
@michaelhixson /cc @nbrady-techempower Thanks! Yes, please don't merge yet. I assumed PRs with failing builds would not be merged but couldn't add labels to PR. Updating PR title to make it easy to spot and will change it back once it ready.

I've been following Kemal's issue and proposed fix https://github.com/kemalcr/kemal/pull/463 and plan to update PR with this fix plus update Crystal to newer 0.25.1 version and make small performance tweaks as well.

**Comment by msmith-techempower** [comment]
I am going to close this pull request for now; feel free to re-open or create a new one once you have addressed the underlying issues.

**Pull request closed without merging**